test/lib/model/test_paraphrase_multilingual.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
wait, but this one is called paraphrase but tests indian_sbert? 😆  maybe wrong file updated?

Reply:
Whoops my copypasta got flipped

Reply:
Good catch. Thanks.